spacify-portal/src/main/java/org/uci/spacifyPortal/services/RoomService.java

```diff
@@ -0,0 +1,40 @@
+package org.uci.spacifyPortal.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
Lets merge this file with the CreateRoomService file. It already creates an entry for the room. We can merge the below two functions in it.